package.json

```diff
@@ -225,6 +225,16 @@
           "description": "Array of keywords that should be removed from the input prior to parsing.",
           "type": "array",
           "default": []
+        },
+        "doxdocgen.generic.useGitUserName": {
+          "description": "Substitute {author} with git config --get user.name.",
+          "type": "boolean",
+          "default": false
+        },
+        "doxdocgen.generic.useGitUserEmail": {
+          "description": "Substitute {email} with git config --get user.email.",
+          "type": "boolean",
+          "default": false
```

Comment:
Please add these values to the README

Reply:
Added by https://github.com/cschlosser/doxdocgen/pull/186/commits/94c96625b0a34db98def47957e33cb47ac93c3b7